tests/raftstore/cluster.rs

```diff
@@ -318,18 +322,27 @@ impl<T: Simulator> Cluster<T> {
             let mut region = self.get_region(key);
             let region_id = region.get_id();
             let req = new_request(region_id, region.take_region_epoch().clone(), reqs.clone());
-            let resp = self.call_command_on_leader(region_id, req, timeout).unwrap();
-            if resp.get_header().has_error() {
-                if self.refresh_leader_if_needed(&resp, region_id) {
-                    warn!("seems leader changed, let's retry");
-                    continue;
-                } else if try_cnt == 1 && resp.get_header().get_error().has_stale_epoch() {
-                    warn!("seems split, let's retry");
-                    try_cnt += 1;
+            let result = self.call_command_on_leader(region_id, req, timeout);
+            match result {
+                Ok(resp) => {
+                    if resp.get_header().has_error() {
+                        if self.refresh_leader_if_needed(&resp, region_id) {
+                            warn!("seems leader changed, let's retry");
+                            continue;
+                        } else if try_cnt <= 5 && resp.get_header().get_error().has_stale_epoch() {
+                            warn!("seems split, let's retry");
+                            try_cnt += 1;
+                            continue;
+                        }
+                    }
+                    return resp;
+                }
+                Err(Error::Timeout(_)) => {
```

Comment:
> it timeout always happen, the test cases won't pass... STILL we'll back again and fix CI
so...seems that it don't matter loop here

Please consider all corner cases, if this happen, CI will wait for a long time and then terminate.